### ISSUE TITLE
Ant Workers hold secondary weapons weirdly

### ISSUE THREAD

**Issue by BugmanBugman** [itsmeow/GoGRedux#1]
![2020-05-20_00 55 07](https://user-images.githubusercontent.com/40730042/82506268-f1d8b480-9ac4-11ea-948b-54099416b96e.png)
me and my friends were spawning in mobs to test them out and found they use secondary weapons oddly

**Comment by itsmeow** [comment]
Yeah was having some issues with this. I'll see next update round if I can improve it at all but that was my best attempt at the time. Ant worker arms are a bit weird
